content/en/devlog/tweasel-update-4/index.md

```diff
@@ -0,0 +1,70 @@
+
+  But sometimes, such documentation is not available. We have a [hierarchy of possible justifications](https://trackers.tweasel.org/research) that we go through in those cases: SDK source code, obvious property names, observed values match known device parameters, obvious observed values, experiments.
+
+  So far, I have produced documentation for the following trackers/adapters, adding missing properties and adapters along the way (though none of them are merged or released yet): [Adjust](https://github.com/tweaselORG/TrackHAR/issues/26) ([PR](https://github.com/tweaselORG/TrackHAR/pull/28)), [branch.io v1](https://github.com/tweaselORG/TrackHAR/issues/30) ([PR](https://github.com/tweaselORG/TrackHAR/pull/31)), [Bugsnag](https://github.com/tweaselORG/TrackHAR/pull/51), [Facebook](https://github.com/tweaselORG/TrackHAR/issues/27) ([PR](https://github.com/tweaselORG/TrackHAR/pull/35)), [ioam.de](https://github.com/tweaselORG/TrackHAR/issues/38) ([PR](https://github.com/tweaselORG/TrackHAR/pull/43)), [Microsoft App Center](https://github.com/tweaselORG/TrackHAR/issues/44) ([PR](https://github.com/tweaselORG/TrackHAR/pull/47)), [OneSignal](https://github.com/tweaselORG/TrackHAR/issues/49) ([PR](https://github.com/tweaselORG/TrackHAR/pull/50)), [singular.net](https://github.com/tweaselORG/TrackHAR/issues/16) ([PR](https://github.com/tweaselORG/TrackHAR/pull/24))
+* When I started the adapter work, we only had a very basic idea of how we actually wanted to document all this but I decided to just get my feet wet and gain a better understanding of the problem as I worked on it. That sparked some very [in-depth discussions](https://github.com/tweaselORG/meta/issues/3#issuecomment-1682038715). To summarize the main conclusions:
+    * Where should the documentation be stored (in pads hosted somewhere, in GitHub issues, in Markdown files in some repo, …)? We decided to keep the documentation as [Markdown files in the TrackHAR repo](https://github.com/tweaselORG/TrackHAR/pull/32), such that each page gets its own folder and each property its own file. We glue the sections together using Hugo in the `tracker-wiki` repo (which Lorenz has already [implemented](https://github.com/tweaselORG/tracker-wiki/pull/7)). This way, we can easily reference the reasoning for each property and generate links to the final documentation.
```

Comment:
```suggestion
    * Where should the documentation be stored (in pads hosted somewhere, in GitHub issues, in Markdown files in some repo, …)?  
We decided to keep the documentation as [Markdown files in the TrackHAR repo](https://github.com/tweaselORG/TrackHAR/pull/32), such that each page gets its own folder and each property its own file. We glue the sections together using Hugo in the `tracker-wiki` repo (which Lorenz has already [implemented](https://github.com/tweaselORG/tracker-wiki/pull/7)). This way, we can easily reference the reasoning for each property and generate links to the final documentation.
```

I think the questions look weird in the text, so we should treat them as some kind of heading.